twill-core/src/main/java/org/apache/twill/kafka/client/KafkaConsumer.java

```diff
@@ -33,14 +33,16 @@
 
     /**
      * Invoked when new messages is available.
+     * @param startOffset Offset of the first {@link FetchedMessage} in the iterator of new messages.
```

Comment:
If one specified `-1L` or `-2L`, but no message is consumed in the method, then `-1L` or `-2L` still has to be returned